BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -299,10 +299,10 @@ class CbrainTask::<%= name %> < ClusterTask
 
         self.addlog("Saving result file #{path}")
         name = File.basename(path)
-
-        output = safe_userfile_find_or_new((
-          File.directory?(path) ? FileCollection : Userfile.suggested_file_type(name)
-        ), :name => name)
+        
+	userfile_class = Userfile.suggested_file_type(name)
+        userfile_class ||= ( File.directory?(path) ? FileCollection : SingleFile )
+        output = safe_userfile_find_or_new(userfile_class, :name => name)
```

Comment:
On the LORIS team we've gotten rid of such issues (and the need to point them out) by having Travis run a static analysis tool that checks for style issues like inconsistent whitespace and fails the build if it finds them.

I'm sure Ruby must have some kind of linter that you can use.